src/app/dataset-view/additional-components/dataset-settings-component/tabs/webhooks/create-subscription-modal/create-subscription-modal.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="d-flex p-4">
+    <div class="flex-grow-1 ms-2">
+        <span class="fs-5 fw-bold">{{
+            subscriptionData ? "Webhook subscription details" : "Create subscription"
```

Comment:
If this is both create/edit form simultaneously, then rename the component, as I got confused.

Reply:
Renamed to `CreateEditSubscriptionModalComponent`